content/courses/onchain-development/anchor-cpi.md

```diff
@@ -467,20 +476,29 @@ pub struct AddMovieReview<'info> {
     )]
     pub token_account: Account<'info, TokenAccount>,
     pub associated_token_program: Program<'info, AssociatedToken>,
-    pub rent: Sysvar<'info, Rent>
 }
 ```
 
 Again, some of the above constraints may be unfamiliar to you. The
 `associated_token::mint` and `associated_token::authority` constraints along
 with the `init_if_needed` constraint ensures that if the account has not already
 been initialized, it will be initialized as an associated token account for the
-specified mint and authority.
+specified mint and authority. Also, the payer for the costs related with the
+account initialiaziation will be set under the constraint `payer`.
+
+If you're unfamiliar with the `INIT_SPACE` constant used for the `movie_review`
+account space allocation, please refer to the `solution-pdas` branch that is
```

Comment:
could you add a link to the specific place where this is talked about?

Reply:
Agreed. Link added when referring the `solution-pdas` branch